dataprep/connector/connector.py

```diff
@@ -145,74 +119,10 @@ async def query(  # pylint: disable=too-many-locals
 
         return await self._query_imp(table, where, _auth=_auth, _q=_q, _count=_count)
 
-    @property
-    def table_names(self) -> List[str]:
-        """
-        Return all the names of the available tables in a list.
-
-        Note
-        ----
-        We abstract each website as a database containing several tables.
-        For example in Spotify, we have artist and album table.
-        """
-        return list(self._impdb.tables.keys())
-
     def info(self) -> None:
         """Show the basic information and provide guidance for users
         to issue queries."""
-
-        # get info
-        tbs: Dict[str, Any] = {}
-        for cur_table in self._impdb.tables:
-            table_config_content: ConfigDef = self._impdb.tables[cur_table].config
-            params_required = []
-            params_optional = []
-            example_query_fields = []
-            count = 1
-            for k, val in table_config_content.request.params.items():
-                if isinstance(val, bool) and val:
-                    params_required.append(k)
-                    example_query_fields.append(f"""{k}="word{count}\"""")
-                    count += 1
-                elif isinstance(val, bool):
-                    params_optional.append(k)
-            tbs[cur_table] = {}
-            tbs[cur_table]["required_params"] = params_required
-            tbs[cur_table]["optional_params"] = params_optional
-            tbs[cur_table]["joined_query_fields"] = example_query_fields
-
-        # show table info
-        print(INFO_TEMPLATE.render(ntables=len(self.table_names), dbname=self._impdb.name, tbs=tbs))
-
-    def show_schema(self, table_name: str) -> pd.DataFrame:
-        """This method shows the schema of the table that will be returned,
-        so that the user knows what information to expect.
-
-        Parameters
-        ----------
-        table_name
-            The table name.
-
-        Returns
-        -------
-        pd.DataFrame
-            The returned data's schema.
-
-        Note
-        ----
-        The schema is defined in the configuration file.
-        The user can either use the default one or change it by editing the configuration file.
-        """
-        print(f"table: {table_name}")
-        table_config_content: ConfigDef = self._impdb.tables[table_name].config
-        schema = table_config_content.response.schema_
-        new_schema_dict: Dict[str, List[Any]] = {}
-        new_schema_dict["column_name"] = []
-        new_schema_dict["data_type"] = []
-        for k in schema.keys():
-            new_schema_dict["column_name"].append(k)
-            new_schema_dict["data_type"].append(schema[k].type)
-        return pd.DataFrame.from_dict(new_schema_dict)
+        info(self._impdb.name, self._update)
```

Comment:
Why do we need to pass `update` to info? If Connector is created with update = True, then the config is already updated. Otherwise, this is False anyway.

Reply:
Currently the default for the update parameter of the info function is true. Should I change that to false and remove self._update as an argument for this call of the info function?

Will this solve the issue?

Reply:
Yes, please. Thanks!